src/services/jwt.service.js

```
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '5m',
+  });
```

This line calls `jwt.sign(user, process.env.JWT_KEY, ...)` directly. If `process.env.JWT_KEY` is missing or empty `jwt.sign` may throw or produce incorrect tokens. Validate the presence of `JWT_KEY` at application startup and fail fast with a clear error if it's not set.